CHANGELOG.rst

```diff
@@ -4,6 +4,13 @@ Openvswitch Collection Release Notes
 
 .. contents:: Topics
 
+v2.0.1
```

Comment:
I wonder if we should include a note about 2.0.0 release, maybe something around its broken.

Reply:
now that we've the tagged 2.0.0 in place, so it won't be broken. Do we need to specifically mention about 2.0.0 as we have mentioned in the comment  that it wasn't intended and result of a tag mistake

Reply:
We should have a 2.0.0 entry in the changelog, even if it says do not use or it is broken.

Trying to avoid confusion when people look at changelog, and see 2.0.0 on galaxy / AH.

Reply:
ack, updated